TickingHeartRate/runTickingHeartRateReplay.py

```diff
@@ -1,34 +1,35 @@
-from deephaven.TableTools import readHeaderlessCsv
+from deephaven import csv
```

Comment:
I don't think this is the right import. Shouldn't it be

```suggestion
from deephaven import read_csv
```

Reply:
I think either syntax is ok.